gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -26,9 +26,34 @@ pub async fn run_webserver(
     gateway: Gateway,
     task_group: &mut TaskGroup,
 ) -> axum::response::Result<()> {
-    let (routes, admin_routes) = if let Some(gateway_config) = config {
+    let routes = v1_routes(config, gateway.clone());
+    let api_v1 = Router::new()
+        .nest("/v1", routes.clone())
+        // Backwards compatibility: Continue supporting gateway APIs without versioning
```

Comment:
In the client, perhaps we could add a warning message that is printed whenever the non-versioned API is used? The worry I have with supporting backwards compatibility is introducing unexpected changes to the non-versioned API

Reply:
A new gateway deployed will register it's versioned API so all clients in the fed never get to use it's non-versioned endpoints. Non-versioned api's are likely to be (mis)used by gateway UIs